docs/ndcube.rst

```diff
@@ -91,10 +91,9 @@ pixel units giving the length of each dimension in the
 `~ndcube.NDCube` while `~ndcube.NDCube.world_axis_physical_types`
 returns an iterable of strings denoting the type of physical property
 represented by each axis.  The axis names are in accordance with the
-International Virtual Observatory Alliance (IVOA) `UCD1+ controlled
-vocabulary <
-http://www.ivoa.net/documents/REC/UCD/UCDlist-20070402.html>`. Here 
-the shape and axis types are given in data order, not WCS order. 
+International Virtual Observatory Alliance (IVOA)
+`UCD1+ controlled vocabulary <http://www.ivoa.net/documents/REC/UCD/UCDlist-20070402.html>`.
```

Comment:
does this not need an `_` after the last backtick?

Reply:
As in ````UCD1+ controlled vocabulary <http://www.ivoa.net/documents/REC/UCD/UCDlist-20070402.html>`_```
?